Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs

```diff
@@ -45,11 +45,10 @@ public static void Reset()
         /// </summary>
         public static void Start()
         {
-            Process process = new Process { StartInfo = StartIis };
-            process.Start();
-            process.WaitForExit();
+            Stop();
```

Comment:
Good catch. Thank you. I misread. I thought start is doing stop and than start of the same process.

---
In reply to: [73772634](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/115#discussion_r73772634) [](ancestors = 73772634)